app/src/main/java/com/daclink/drew/sp22/cst438_project01_starter/User.java

```diff
@@ -9,9 +9,9 @@
     private String firstName;
     private String lastName;
     private String location;
-    @PrimaryKey(autoGenerate = true)
+    @PrimaryKey
     @NonNull
-    private Integer uid;
+    private String username;
```

Comment:
Hey, I think this might give you issues because the primary key that we generate i.e (@PrimaryKey(autoGenerate = true)) has to be stored in an int variable. That int variable has to be added before you add your username variable. 
Maybe this: 
private Int uid; (maybe don't delete );
private String username;